CHANGELOG.md

```diff
@@ -1,60 +1,106 @@
+## 2.2.0 (March XX, 2022)
+
+NOTES:
+
+* resource/local_file: Argument `sensitive_content` is `Deprecated`; please use the new resource `local_sensitive_file`.
+
+FEATURES:
+
+* **New data-source/local_sensitive_file**: Similar to the data source `local_file`, but `content` and `content_base64` attributes
+  are marked as _sensitive_.
+  ([#101](https://github.com/hashicorp/terraform-provider-local/pull/101) and
+  [#106](https://github.com/hashicorp/terraform-provider-local/pull/106))
+* **New resource/local_sensitive_file**: Similar to the resource `local_file`, but `content` and `content_base64` arguments
+  are marked as _sensitive_.
+  ([#106](https://github.com/hashicorp/terraform-provider-local/pull/106))
+
 ## 2.1.0 (February 19, 2021)
 
-Binary releases of this provider now include the darwin-arm64 platform. This version contains no further changes.
+NOTES:
+
+* Binary releases of this provider now include the` darwin-arm64` platform.
+* This version contains no further changes.
 
 ## 2.0.0 (October 14, 2020)
 
-Binary releases of this provider now include the linux-arm64 platform.
+NOTES:
+
+* Binary releases of this provider now include the `linux-arm64` platform.
 
 BREAKING CHANGES:
 
-* Upgrade to version 2 of the Terraform Plugin SDK, which drops support for Terraform 0.11. This provider will continue to work as expected for users of Terraform 0.11, which will not download the new version. ([#42](https://github.com/terraform-providers/terraform-provider-local/issues/42))
+* Upgrade to version 2 of the Terraform Plugin SDK, which drops support for Terraform 0.11.
+  This provider will continue to work as expected for users of Terraform 0.11, which will not download the new version.
+  ([#42](https://github.com/terraform-providers/terraform-provider-local/issues/42))
```

Comment:
Aside: Automation tooling like `go-changelog` won't implement line breaks, so not sure if there's benefit to changing the old entries.